VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -3833,6 +3833,33 @@ public void TestPregnancyCodes()
         }
 
         [Fact]
+        public void TestEducationLevelObs()
+        {
+            var testRecord = new DeathRecord();
+            Composition composition = testRecord.ReturnCompositionForTesting();
+            bool covered = false;
+
+            int[] beforeCounts = new int[composition.Section.Count];
+
+            for(int i=0;i<beforeCounts.Length;i++)
+            {
+                beforeCounts[i] = composition.Section[i].Entry.Count;
+            }
+
+            testRecord.EducationLevelHelper = "HS";
+
+            // Verify we added a section entry with the above statement
+            for (int i = 0; i < beforeCounts.Length; i++)
+            {
+                if(beforeCounts[i]+1 == composition.Section[i].Entry.Count)
```

Comment:
Are you saying at least one section[i] has an extra entry?  If so you should "break" on this condition instead of continuing looping.

Also, you should check the section entry and make sure it's for DecedentDemographics by name, and not some other name.